src/qt/rpcconsole.cpp

```diff
@@ -436,10 +436,11 @@ void RPCConsole::clear()
     }
 
     // Set default style sheet
+    QFontDatabase::addApplicationFont(":/fonts/Inconsolata-Regular.ttf");
     ui->messagesWidget->document()->setDefaultStyleSheet(
                 "table { }"
-                "td.time { color: #808080; padding-top: 3px; } "
-                "td.message { font-family: Monospace; font-size: 12px; } "
+                "td.time { color: #808080; valign: bottom;} "
+                "td.message { valign: bottom; } "
                 "td.cmd-request { color: #006060; } "
                 "td.cmd-error { color: red; } "
                 "b { color: #006060; } "
```

Comment:
I think you have to put the font in the stylesheet here, since you took it out of the system-wide version.. We are sort of back to where I was with my PR... :)

The console is showing (the very pretty) proportional font...
![image](https://user-images.githubusercontent.com/7529186/95006812-4729e400-05d6-11eb-8378-6607cdaac980.png)

Reply:
If you load the light or dark themes it acts as it should, the native style does not.
The font is loaded when the RPC console is called, I believe the native style is just missing some needed elements.

Reply:
At least that's the way it reacting here.

Reply:
NO. That IS the dark theme. Look carefully. That is a proportional font. Please see my screenshot with the correct font.